api-reference/_hidden/dxActionSheetItem/type.md

```diff
@@ -9,4 +9,6 @@ default: 'normal'
 Specifies the [type](/api-reference/10%20UI%20Components/dxButton/1%20Configuration/type.md '/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type') of the button that represents an action sheet item.
```

Comment:
I'd like to avoid using "represents" if possible. Can we change it to "is"?

Reply:
Ok

Reply:
Thanks. I added the suggestion below:

```suggestion
Specifies the [type](/api-reference/10%20UI%20Components/dxButton/1%20Configuration/type.md '/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type') of the button that is an action sheet item.
```